tensorflow/lite/tools/evaluation/evaluation_delegate_provider.cc

```diff
@@ -132,6 +132,10 @@ tools::ToolParams DelegateProviders::GetAllParams(
     tool_params.Set<int32_t>("num_threads", params.num_threads());
   }
 
+  if (params.has_nnapi_allow_fp16()) {
```

Comment:
These changes are not needed as the flag is already added to the nnapi_delegate_provider.cc